src/components/infinite-scroll/infinite-scroll.ts

```diff
@@ -218,6 +218,15 @@ export class InfiniteScroll {
   }
 
   /**
+  * Pass a promise inside `waitFor()` within the `infinite` output event handler in order to
+  * change state of infiniteScroll to "complete"
```

Comment:
Do you want me to add example at the top of the component inside `@usage` section or at the top of the method?